pkg/auth/oidc/oidc.go

```diff
@@ -44,19 +49,37 @@ type OpenIDStrategy struct {
 
 // NewOpenIDStrategy returns OpenId auth strategy
 func NewOpenIDStrategy(options *auth.Options) (*OpenIDStrategy, *auth.Options, error) {
-	httpClient := util.BuildHTTPClient(options.SSLDisabled)
+	var httpClient *http.Client
+	var err error
+	if len(options.Cert) > 0 && len(options.Key) > 0 {
+		if httpClient, err = util.BuildHTTPClientWithCert(options.Cert, options.Key); err != nil {
```

Comment:
you're checking it twice in the func,
store it in variable -> mtlsEnabled := len(options.Cert) > 0 && len(options.Key) > 0

Reply:
done